test/testdata/generate.sh

```diff
@@ -123,12 +119,9 @@ build_kb
 
 # Project version 2 uses plugin go/v2 (default).
 scaffold_test_project project-v2 --project-version=2
```

Comment:
Should we remove this one as well?

Reply:
I do not think so.
Why? we need to ensure in the reviews that changes are not made in this scaffold. 
By keeping it, we can easily check that.

Reply:
Makes sense